toolchains/esp32/sdkconfig

```diff
@@ -1368,6 +1362,8 @@ CONFIG_CONSOLE_UART_DEFAULT=y
 CONFIG_CONSOLE_UART=y
 CONFIG_CONSOLE_UART_NUM=0
 CONFIG_CONSOLE_UART_BAUDRATE=115200
+# CONFIG_INT_WDT is not set
```

Comment:
This pruning is automatically done by the build process. Didn't catch it in my last commit.